### PR TITLE
Replace "PROJECT" with project object

### LINKED ISSUE #6745: Hope that Translates API returns language code of each project
Thanks for hacking such a useful product! 👏

Especially the public API is awesome for who's motivated to automate workflow.
With help from your public API v3, [I successfully replaced puppeteer](https://github.com/KengoTODA/rtd-bot/pull/400) that makes my bot really fat and unstable.

Here I have a request for future development (API v4?), I hope it helps your hack and your community.

----

I have [code with Translatest API v2 to make a list of related projects](https://github.com/KengoTODA/rtd-bot/blob/2888744ff27b41cb71ad33a42ccec9ca72a195d3/src/rtd.ts#L33-L50). In short I want to get information like `{ translations: Array<{slug: string, languageCode: string}> }` from the API.

API v2 supported it, however, API v3 does not respond `languageCode` data, so I need to invoke [project details API](https://docs.readthedocs.io/en/stable/api/v3.html#project-details) one by one.

Not sure it's an intentional change. I personally prefer the behavior of v2, so I wish the next API will support it.

### PR DESCRIPTION
**Pull request by KengoTODA**
close #6745

To make the document intuitive, replace `"PROJECT"` with project object.

### CODE REVIEW

**Comment by KengoTODA** [comment]
OK but can I use `:http:get:`/api/v3/projects/(string:project_slug)/` instead of v2?

https://docs.readthedocs.io/en/stable/api/v3.html#get--api-v3-projects-(string-project_slug)-

**Comment by stsewd** [comment]
> OK but can I use :http:get:/api/v3/projects/(string:project_slug)/` instead of v2?

Yes, my mistake!

File: docs/api/v3.rst [hunk 1]
```diff
@@ -103,7 +103,11 @@ Projects list
             "count": 25,
             "next": "/api/v3/projects/?limit=10&offset=10",
             "previous": null,
-            "results": ["PROJECT"]
+            "results": [{
+                "id": 12345,
+                "name": "Pip",
+                ...
+            }]
```
Comment (humitos) [inline docs/api/v3.rst:110]: Is this more clear in the end? I'm not sure that mentioning just 2 fields will make people realize that it's a project (with all its fields) in all the endpoints. What are we using in the rest of the docs? We should be consistent through the whole document.

Also, maybe we want to be more explicit here saying something like "Project object" or "List of Project objects" maybe with a link to the Project Details section if possible.

File: docs/api/v3.rst [hunk 2]
```diff
@@ -863,9 +917,63 @@ Translations listing
...
+                    "subprojects": "/api/v3/projects/pip/subprojects/",
+                    "superproject": "/api/v3/projects/pip/superproject/",
+                    "redirects": "/api/v3/projects/pip/redirects/",
+                    "translations": "/api/v3/projects/pip/translations/"
+                }
+            }]
         }
 
+        The ``results`` in response is an array of project data, which is same to the `Project API <https://docs.readthedocs.io/en/stable/api/v3.html#get--api-v3-projects-(string-project_slug)->`_.
+
```
Comment (stsewd) [inline docs/api/v3.rst:976]: The link should be a ref role, not a hardcoded url

Similar to `` :http:get:`/api/v2/project/(int:id)/` ``

Comment (KengoTODA) [reply docs/api/v3.rst:976]: I'm not so sure about this `http` directive, and cannot find existing usage by searching `<http:[^/]` in project. Let me push untested code to check the behavior.

**Pull request merged**